packages/apputils-extension/style/contextualhelp.css

```
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+:root {
+  --jp-private-shortcuts-table-padding-vertical: 9px;
+  --jp-private-shortcuts-table-padding-right: 14px;
+  --jp-private-shortcuts-key-padding-horizontal: 5px;
+  --jp-private-shortcuts-key-padding-vertical: 3px;
+}
+
+tr,
+td {
+  padding-bottom: var(--jp-private-shortcuts-table-padding-vertical);
+  padding-top: var(--jp-private-shortcuts-table-padding-vertical);
+  padding-right: var(--jp-private-shortcuts-table-padding-right);
+}
+
+.jp-ContextualShortcut-TableRow {
+  font-size: var(--jp-ui-font-size1);
+  font-family: var(--jp-ui-font-family);
+}
+
+.jp-ContextualShortcut-TableItem {
+  margin-left: auto;
+  margin-right: auto;
+  vertical-align: center;
+  background-color: var(--jp-layout-color0);
+  color: var(--jp-inverse-layout-color0);
+  font-size: var(--jp-ui-font-size0);
+  font-family: var(--jp-code-font-family);
+}
+
+.jp-ContextualShortcut-TableHeader {
+  text-align: left;
+}
+
+.jp-ContextualShortcut-TableLastRow {
+  height: '2em';
+}
+
+.jp-ContextualShortcut-TableItem .jp-ContextualShortcut-Key {
```

Do we need to combine the selector? If not let's drop the `TableItem` part

```suggestion
.jp-ContextualShortcut-Key {
```